projects/packages/connection/src/class-manager.php

```diff
@@ -21,38 +20,6 @@
  * and Jetpack.
  */
 class Manager {
-
-	const SECRETS_MISSING        = 'secrets_missing';
-	const SECRETS_EXPIRED        = 'secrets_expired';
-	const SECRETS_OPTION_NAME    = 'jetpack_secrets';
-	const MAGIC_NORMAL_TOKEN_KEY = ';normal;';
-
-	/**
-	 * Constant used to fetch the master user token. Deprecated.
-	 *
-	 * @deprecated 9.0.0
-	 * @see Manager::CONNECTION_OWNER
-	 * @var boolean
-	 */
-	const JETPACK_MASTER_USER = true; //phpcs:ignore Jetpack.Constants.MasterUserConstant.ShouldNotBeUsed
-
-	/**
-	 * For internal use only. If you need to get the connection owner, use the provided methods
-	 * get_connection_owner_id, get_connection_owner and is_connection_owner
-	 *
-	 * @todo Add private visibility once PHP 7.1 is the minimum supported verion.
-	 *
-	 * @var boolean
-	 */
-	const CONNECTION_OWNER = true;
```

Comment:
Seems like this removal is what's failing the tests https://github.com/Automattic/jetpack/pull/18812/checks?check_run_id=1891048859#step:14:726

Reply:
Thanks for the tip! Just pushed a change to fix this.